libs/deal-ticket/src/components/market-selector.tsx

```diff
@@ -116,6 +116,7 @@ export const MarketSelector = ({ market, setMarket, ItemRenderer }: Props) => {
       market: MarketNames_markets,
       index: number
     ) => {
+      event.preventDefault();
```

Comment:
suggestion (non-blocking / a11y): `preventDefault` here will block normal `[tab]`bing behaviour - consider moving it to the 'allowed keys' if needed

Reply:
Handling Tab press on suggest list has been added